packages/babel-plugin-ember-test-metadata/get-relative-paths.js

```diff
@@ -14,7 +14,7 @@ function _getRelativeProjectPath(pathSegments, projectDir, projectRoot) {
 
 function _getRelativePathForClassic(filePath, packageName, projectRoot) {
   const projectDir = _getNormalizedPackageDir(packageName);
-  const pathSegments = filePath.split(sep);
+  const pathSegments = filePath.split(sep).filter(segment => segment !== 'dummy')
```

Comment:
Missing ending semicolon. Also, let's wrap the `segment` param in parentheses to be consistent with the rest of the code base. Not sure why these aren't getting caught by ESLint.